docker/entry.sh

```diff
@@ -4,7 +4,7 @@ sa_dir=/var/run/secrets/kubernetes.io/serviceaccount
 
 kc_args="--server=https://kubernetes.default.svc.cluster.local --token=$(cat $sa_dir/token) --certificate-authority=$sa_dir/ca.crt"
 
-kubectl $kc_args create -f manifests/
+kubectl $kc_args apply -f manifests/
 
 if [ $? -ne 0 ]; then
     echo "Failed to apply manifest/ folder"
```

Comment:
s/manifest/manifests
`Failed to apply manifests/` looks cleaner to me.

Reply:
Ack